src/store/index.js

```diff
@@ -1,12 +1,15 @@
-import { createStore } from 'vuex'
+// stores/counter.js
+import { defineStore } from 'pinia'
 
-export default createStore({
-  state: {
-  },
-  mutations: {
+export const useCounterStore = defineStore('counter', {
+  state: () => {
```

Comment:
What is count? This doesn't seem to describe anything specific. Unless you're counting the times that the app was connected to? It could still be more descriptive in my opinion if that's the case.